package.json

```diff
@@ -4,21 +4,22 @@
   "description": "It's cool when users own their content.",
   "main": "index.js",
   "scripts": {
-    "build": "./node_modules/.bin/gulp",
+    "build": "gulp",
```

Comment:
Awesome, I wasn't aware of that. Maybe you could issue that change as its own PR :)